packages/trpc/server/routers/viewer/teams/inviteMember/inviteMember.handler.ts

```diff
@@ -93,14 +98,26 @@ export const inviteMemberHandler = async ({ ctx, input }: InviteMemberOptions) =
     { emails: [], usernames: [] } as { emails: string[]; usernames: string[] }
   );
 
+  // New Users can always be invited
   const newUsersEmailsOrUsernames = usernameOrEmailsToInvite.filter(
     (usernameOrEmail) =>
       !existingUsersEmailsAndUsernames.emails.includes(usernameOrEmail) &&
       !existingUsersEmailsAndUsernames.usernames.includes(usernameOrEmail)
   );
 
-  log.debug(
-    "inviteMemberHandler",
+  if (
+    !beSilentAboutErrors &&
+    existingUsersWithMemberships.length &&
+    !existingUsersWithMembershipsThatNeedToBeInvited.length
+  ) {
+    throw new TRPCError({
```

Comment:
Allow showing the error in case of a single invite so that the user is informed of the failure.